addons/account/models/res_partner_bank.py

```diff
@@ -48,10 +41,14 @@ class ResPartnerBank(models.Model):
              'To protect yourself, always verify new bank account numbers, preferably by calling the vendor, as phishing '
              'usually happens when their emails are compromised. Once verified, you can activate the ability to send money.'
     )
-    currency_id = fields.Many2one(tracking=True)
     lock_trust_fields = fields.Boolean(compute='_compute_lock_trust_fields')
     duplicate_bank_partner_ids = fields.Many2many('res.partner', compute="_compute_duplicate_bank_partner_ids")
 
+    def _get_fiscal_country_codes(self):
+        return ','.join(self.env.companies.mapped('account_fiscal_country_id.code'))
+
+    fiscal_country_codes = fields.Char(store=False, default=_get_fiscal_country_codes)
```

Comment:
where is this used? 
(a hidden file perhaps 👀 )

Reply:
should we not keep it there? seems to be the only l10n using it

Reply:
I don't think we should move it since it's only used by MX
I have also seen l10n_id add fields to the `res.partner.bank` form but it uses the country code of the same model.
Could we not do the same for mexico and remove the `fiscal_country_codes` field